app/views/govuk_publishing_components/component_guide/show.html.erb

```diff
@@ -24,7 +24,7 @@
           </div>
         </div>
       <% end %>
-      <p class="govuk-body"><%= link_to "Search for usage of this component on GitHub", @component_doc.github_search_url, class: "govuk-link" %>.</p>
+      <p class="govuk-body"><%= link_to "Search for usage of this component on GitHub.", @component_doc.github_search_url, class: "govuk-link" %>.</p>
```

Comment:
Usual convention is to not include prior or following punctuation in link text, like the full stop here. Also there's already a full stop at the end of the sentence, so I'm confused why this has been added. Currently looks like this:

![Screenshot 2021-10-06 at 16 36 26](https://user-images.githubusercontent.com/861310/136236381-2930e443-3460-44f4-8483-1ec012e50706.png)